webpack.config.js

```diff
@@ -11,9 +11,7 @@ const path = require("path");
 var Visualizer = require("webpack-visualizer-plugin");
 const ObjectRestSpreadPlugin = require("@sucrase/webpack-object-rest-spread-plugin");
 
-function isDevelopment() {
-  return process.env.NODE_ENV !== "production";
-}
+const isProduction = process.env.NODE_ENV === "production";
```

Comment:
I **_think_** this check can just be a constant. Let me know if that's wrong

Reply:
yep